components/automate-ui/src/styles.scss

```diff
@@ -196,3 +196,49 @@ chef-loading-spinner[fixed] {
 input {
   line-height: normal;
 }
+
```

Comment:
Angular Material requires a lot of over rides, but doesn't really seem to have a good place to target globally outside of the styles.css file.  Any suggestions on a better location for something like this.  Will get crowded after adding several more components.